src/core/qgsrenderedlayerstatistics.h

```diff
@@ -0,0 +1,133 @@
+/***************************************************************************
+    qgsrenderedlayerstatistics.h
+    ----------------
+    copyright            : (C) 2024 by Jean Felder
+    email                : jean dot felder at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSRENDEREDLAYERSTATISTICS_H
+#define QGSRENDEREDLAYERSTATISTICS_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgsrendereditemdetails.h"
+
+#include <limits>
+
+
+/**
+ * \ingroup core
+ * \brief Contains computed statistics for a layer render.
+ * \since QGIS 3.42
+ */
+class CORE_EXPORT QgsRenderedLayerStatistics: public QgsRenderedItemDetails
+{
+  public:
+
+    /**
+     * Constructor for QgsRenderedLayerStatistics from a list of \a minimum and \a maximum.
+     */
+    QgsRenderedLayerStatistics( const QString &layerId, QList<double> minimum, QList<double> maximum );
```

Comment:
```suggestion
    QgsRenderedLayerStatistics( const QString &layerId, const QList<double>& minimum, const QList<double>& maximum );
```

Reply:
What's this mean though? Can you elaborate in the docs when this would be used and what the list values represent?